### PR TITLE
Add PHP 8.3, 8.4 to CI

### PR DESCRIPTION
**Pull request by andreybolonin**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['8.0', '8.1', '8.2']
+        php: ['8.0', '8.1', '8.2', '8.3', '8.4']
```
Comment (bocharsky-bw) [inline .github/workflows/ci.yml:13]: I think it's time to drop 8.0 from that list.

**Review by bocharsky-bw — COMMENTED** [review]
Could you also fix PHP CS Fixer?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (andreybolonin) [reply .github/workflows/ci.yml]: done

Comment (andreybolonin) [reply .github/workflows/ci.yml]: @rvanlaak done

**Review by bocharsky-bw — APPROVED** [review]
Great, thank you!

**Pull request merged**